transform/transformers/ImageTransformer.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python
+#   coding: UTF-8
+
+import argparse
+import datetime
+import dateutil.parser
+import glob
+from io import BytesIO
+import itertools
+import json
+import logging
+import os.path
+import shutil
+import subprocess
+import sys
+import zipfile
+
+from requests.packages.urllib3.exceptions import MaxRetryError
+
+try:
+    from PDFTransformer import PDFTransformer
+except ImportError:
+    from .PDFTransformer import PDFTransformer
```

Comment:
Surely we can dispense with the `try` block here, as we know we have a `PDFTransformer` module?